src/Commands.hs

```diff
@@ -293,23 +293,203 @@ commandListBindings ctx args =
               putStrLn "\nGlobal environment:\n"
               putStrLn (prettyEnvironment (contextGlobalEnv ctx))
               putStrLn ""
-              return (ctx, dynamicNil)
+              pure (ctx, dynamicNil)
+
+-- | Command for printing help.
```

Comment:
Why was this moved back here? I think `(help)` should be a primitive so that you can access the chapters without quoting their names.

Reply:
Oh, I messed it with the rebase. Should be fixed now.